models/us_fdic_bankfind/code/download_and_clean.py

```
@@ -0,0 +1,266 @@
+    if args.quarters:
+        quarters = quarters[-args.quarters :]
+    print(f"quarters to build: {len(quarters)}", flush=True)
+
+    manifest, done = [], 0
+    with ProcessPoolExecutor(max_workers=args.workers) as pool:
+        futures = {pool.submit(build_quarter, q): q for q in quarters}
+        for future in as_completed(futures):
+            quarter = futures[future]
+            try:
+                result = future.result()
+            except Exception as error:  # keep going; rerun picks up the gap
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)